### ISSUE TITLE
is this library is going to go obsolete soon? 

### ISSUE THREAD

**Issue by mhamri** [handsontable/formula-parser#120]
I can see you are working on Hyperformual and it's not free anymore for commercial usage. I wonder if I should look into other libraries and go away from this library or this library still get update and fixes.

**Comment by AMBudnik** [comment]
Hey @mhamri 

We are still using this library in our commercial product. However, we are switching to a new solution soon, so we will not update this library.
However, I can see more than 150 forks, and I believe that some (if not all) are distributed under an MIT license.

**Comment by MartinDawson** [comment]
@AMBudnik I presume it's hyperformula that is what you are switching to.

What is the exact difference between formula-parser and hyperformula in terms of speed and features?

Both traverse the nodes in the graph smartly right using AST trees?

**Comment by AMBudnik** [comment]
@MartinDawson yes, we are switching to HyperFormula. 

We took some time to develop the solution and made sure it meets most of the market's requirements (and our own). What has changed? A lot I would say.

* Support for efficient large dataset calculations
* Better responsibility for multiple datasheets (especially for tables with 10 ^4 number of cells)
* Better calculation precision - without the JavaScript 0.1+0.2=0.300000004 issue
* Higher compatibility with what's implemented in Excel'em and Google Sheets
* Larger API
* Working on virtual sheets
* Easier data visualization
* Written in TypeScript
* Created based on an opened standard (compatible with the OpenFormula™ standard)
* Uses GPU acceleration for better performance
* Available in 16 languages
* Support for CRUD operations (move, add/remove columns and rows)
* Support for cross-sheet references
* Support for undo/redo
* Support for copy/cut/paste
* Support for wildcard characters
* Better support for dates
* Types and formats tracking
* Fixed issues in some of the formulas

**Comment by MartinDawson** [comment]
@AMBudnik Awesome. Thanks for the reply, I'm actually a paying user of it in our project and it's great. Can this be added to hyperformula website to make it easier to justify the cost against competitors and this old version? It would be very useful.

Because currently I had to do a lot of comparisons and digging in source code to see if it was worth it at first.

Thanks

**Comment by AMBudnik** [comment]
That's true. If someone would like to switch to HyperFormula, they would need to do some digging. I have mentioned to the team that a list like this would definitely help.

**Comment by MartinDawson** [comment]
@AMBudnik Ah okay so it's for performance like virtualization then, thanks for explaining.

I use a canvas based spreadsheet so presume this feature is not applicable to mine.

Thanks
